docs/source/resources/scyllaclusters/basics.md

```diff
@@ -0,0 +1,123 @@
+# ScyllaClusters
+
+## Introduction
+
+[ScyllaCluster](../../api-reference/groups/scylla.scylladb.com/scyllaclusters.rst) defines a ScyllaDB datacenter and manages the racks within.
+This section aims to make you familiar with how it looks like and how to perform some of the basic configuration or accessing the APIs.
+By no means this is a complete description of what it can do. Please consult our [generated API reference](../../api-reference/groups/scylla.scylladb.com/scyllaclusters.rst) for a complete list of options.
+
+::::{tip}
+You can always see the currently supported API fields (for a particular version installed in your cluster) by running
+:::{code}
+kubectl explain --api-version='scylla.scylladb.com/v1' ScyllaCluster.spec
+:::
+::::
+
+Note that the Kubernetes clusters are only a regional concept, availability-wise they map into a ScyllaDB datacenter.
+To deploy ScyllaDB cluster with multiple datacenter, you have to combine multiple Kubernetes clusters, each running a [ScyllaCluster](../../api-reference/groups/scylla.scylladb.com/scyllaclusters.rst). To learn more, please see [the dedicated multiDC guide](./multidc/multidc.md).
+
+## Creating a ScyllaCluster
+
+Before we go and create the ScyllaCluster, we'll first create our ScyllaDB config file that we'll reference later in the ScyllaCluster definition.
+
+:::{code-block} bash
+:linenos:
+:emphasize-lines: 7
+
+kubectl apply --server-side -f=<<EOF
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: scylladb-config
+data:
+  scylla.yaml: |
+    authenticator: PasswordAuthenticator
+    authorizer: CassandraAuthorizer
+    # Other options
+EOF
+:::
+
+:::{note}
+Some of the ScyllaDB config is also generated by the {{productName}} based on your ScyllaCluster definitions.
```

Comment:
nit
```suggestion
Part of the ScyllaDB config is also generated by the {{productName}} based on your ScyllaCluster definition.
```

Reply:
there are actually different config files later merged - so to me the "part of config"  doesn't fit that well